Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -571,6 +581,24 @@ end
     break
 end
 
+macro zeroExtend32ToWord(r)
+    if JSVALUE64
+        andq 0xffffffff, r
+    end
+end
```

Comment:
I'm nervous about defining this now without seeing how it's used.  Depending on the use, we might want to rely on offlineasm defining some semantics on certain instructions to achieve this.  Using a macro may result in less optimal code.

Can you defer this to the next patch?  I don't think this affects linkage issues.